src/modm/platform/usb/sam/usb.hpp.in

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Same comment here about code generation

Reply:
Yes, in the case of SAMD the code gen is pretty overkill.

Reply:
There's also the issue that when `using namespace modm::platform;` the `Usb` conflicts with the `Usb` from the CMSIS headers, which needs to be fixed.

Reply:
Oh don't get me started on the CMSIS macros (I have another upcoming PR for RTC support that has the same issue...)